packages/react/src/components/DataTable/DataTable.types.ts

```diff
@@ -136,6 +136,17 @@ export interface DataTableProps<TData extends RowData> {
    * @default false
    */
   enableStickyHeader?: boolean;
+  /**
+   * Whether the canExpand icon is visible or not.
+   * @default true
+   */
+  showCanExpandIcon?: boolean;
+  /**
+   * Prop that can be passed to expand icon returns
+   * number of degree with ChevronDown icon.
+   * @default (isExpanded) => -180 : 0
+   */
+  expandIconFunction?: (isExpanded: boolean) => number;
```

Comment:
this is not great, can we switch some isExpanded boolean? and do 0/-180 below in DataTableExpandButton?